src/com/ableton/Pyenv.groovy

```diff
@@ -51,10 +47,14 @@ class Pyenv implements Serializable {
         // Only export PYENV_ROOT if it isn't part of the Jenkins environment
         installCommands.add("export PYENV_ROOT=${pyenvRoot}")
       }
+      installCommands.add("export PATH=${VirtualEnv.posixPath(pyenvRoot)}/bin:\$PATH")
+      if (script.isUnix()) {
+        installCommands += [
+          "eval \"\$(pyenv init --path)\"",
```

Comment:
`eval "$(pyenv init --path)"` prepends `$PYENV_ROOT/shims` to the `PATH` env var. Isn't there a `shims` folder on Windows?

Reply:
Looking at the production node that ran this CI job, it seems so. It's in `F:\pyenv-win\pyenv-win\shims`. Fixed in 24f6bf9.